ar.c

```diff
@@ -143,6 +150,50 @@ ar_touch (const char *name)
     struct file *arfile;
     arfile = enter_file (strcache_add (arname));
     f_mtime (arfile, 0);
+# ifdef _GUARDIAN_TARGET
+    if (!arfile->updated) {
+      int fd;
+      struct file *file = arfile;
+
+      EINTRLOOP (fd, open (arfile->name, O_RDWR | O_CREAT, 0666));
+      if (fd < 0)
+        TOUCH_ERROR ("touch: open: ");
+      else
+        {
```

Comment:
Indent style inconsistent

Reply:
Here or futher up.

Reply:
> 
> 
> Indent style inconsistent.
> Here or further up and down.

The whole project is full of badly formatted code. I'm sure there is a coding standard of some kind, but it is applied very inconsistently. I was preserving existing constructs as formatted so that they could be easily found and removed if this is actually modified in 4.4. You can find the originally bad formatted stuff in `remake.c`.

https://github.com/ituglib/gmake/blob/6bfe5fedafa4ecbe51f8504ff21a93a06f558a6a/remake.c#L1149-L1150

https://github.com/ituglib/gmake/blob/6bfe5fedafa4ecbe51f8504ff21a93a06f558a6a/remake.c#L1168-L1199